spec/models/comment_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'associations' do
+    it 'should be belongs to user' do
+      expect(Comment.reflect_on_association(:user).macro).to eq(:belongs_to)
+      expect(Comment.reflect_on_association(:post).macro).to eq(:belongs_to)
+    end
+  end
+end
```

Comment:
- [ ] You forgot to write unit tests for the `update_comment_count` method you created in the _**app/models/comment.rb**_ file. Kindly write unit tests for this as it ensures that the method works properly. 👍 🖱️ 

#### Hint: You can create some `Comment` instances and ensure that the value of the `comments_counter` is equal to the number of comments created. 🔹